### PR TITLE
Update test coverage

### PR DESCRIPTION
**Pull request by DMaxfield-BDS**
This is not needed for 3.1.0

### CODE REVIEW

File: src/test/java/com/synopsys/integration/jenkins/detect/extensions/tool/AbortDetectAirGapInstallExceptionTest.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+    private final String toolName = "Test Tool Name";
+    private final String reason = "Test Reason";
+    private final String exceptionMessage = String.format("Cannot install Detect AirGap Installation %s because: %s", toolName, reason);
+
+    @Test
+    public void testException() {
+        ToolInstallation mockToolInstallation = Mockito.mock(ToolInstallation.class);
+        Mockito.when(mockToolInstallation.getName()).thenReturn(toolName);
+        AbortDetectAirGapInstallException abortDetectAirGapInstallException = new AbortDetectAirGapInstallException(mockToolInstallation, reason);
```
Comment (rottebds) [inline src/test/java/com/synopsys/integration/jenkins/detect/extensions/tool/AbortDetectAirGapInstallExceptionTest.java:20]: Does this exception ever actually get used in the codebase?

Comment (DMaxfield-BDS) [reply src/test/java/com/synopsys/integration/jenkins/detect/extensions/tool/AbortDetectAirGapInstallExceptionTest.java:20]: It does not. Should we remove it?

Comment (rottebds) [reply src/test/java/com/synopsys/integration/jenkins/detect/extensions/tool/AbortDetectAirGapInstallExceptionTest.java:20]: Yes, absolutely.

Comment (DMaxfield-BDS) [reply src/test/java/com/synopsys/integration/jenkins/detect/extensions/tool/AbortDetectAirGapInstallExceptionTest.java]: Done

**Review by rottebds — APPROVED** [review]

**Pull request merged**